.pre-commit-hooks.yaml

```diff
@@ -1,6 +1,6 @@
 -   id: nbstripout
     name: nbstripout
     description: 'nbstripout: strip output from Jupyter and IPython notebooks'
-    entry: bash -c 'nbstripout "$@"; git add -u' --
+    entry: bash -c 'nbstripout "$@"' --
     language: python
```

Comment:
This is effectively the same as just `nbstripout` iiuc

Reply:
Yeah, in the end I removed the bash also as that made it not work on Windows. At the end of the day I think a plain `entry: nbstripout` works best.

Reply:
Can you please make that change and I'll merge this PR.